hs/test/Tests.hs

```diff
@@ -303,33 +303,33 @@ testEmptyInputSet =
   let
     txbody = Tx
               Set.empty
-              [ TxOut aliceAddr (Coin 1)]
+              [ TxOut aliceAddr (Lovelace 1)]
               Set.empty
-              (Coin 584)
+              (Lovelace 584)
               (Slot 100)
     aliceWit = makeWitness alicePay txbody
     tx = TxWits txbody (Set.fromList [aliceWit])
   in ledgerState [tx] @?=
        Left [ InputSetEmpty
-            , ValueNotConserved (Coin 0) (Coin 585)
+            , ValueNotConserved (Lovelace 0) (Lovelace 585)
             , InsufficientWitnesses]
 
 testFeeTooSmall :: Assertion
 testFeeTooSmall =
   let
     tx = aliceGivesBobLovelace
            (TxIn genesisId 0)
-           (Coin 3000) (Coin 1) (Coin 0) (Coin 0)
+           (Lovelace 3000) (Lovelace 1) (Lovelace 0) (Lovelace 0)
            Set.empty (Slot 100)
   in ledgerState [tx] @?=
-       Left [ FeeTooSmall (Coin 538) (Coin 1) ]
+       Left [ FeeTooSmall (Lovelace 550) (Lovelace 1) ]
```

Comment:
our simple transaction size function now produces a larger minfee since the string `Lovelace` is longer than `Coin`.